packages/xdl/src/modules/config.js

```diff
@@ -327,6 +327,19 @@ const expoSdkUniversalModules = [
     libName: 'expo-image-picker',
     sdkVersions: '>=33.0.0',
   },
+  {
+    podName: 'EXInAppPurchases',
+    libName: 'expo-in-app-purchases',
+    sdkVersions: '>=33.0.0',
+    config: {
+      ios: {
+        includeInExpoClient: false,
+      },
+      android: {
+        includeInExpoClient: false,
```

Comment:
If I remember correctly, we add `expo-payments-stripe` to Expo Client on Android. Do we not include it due to some external restriction, or module restriction, or do we want to have feature parity between Clients?

Reply:
Yeah it seems we do. I know that we won't be able to include it on iOS because it is not likely to get approved for the app store. I had assume it would be the case on Android but perhaps not. Perhaps @ide can chime in?

Reply:
IAPs require configuration on Google Play / App Store Connect. Since we are the owners of `host.exp.Exponent`, only we can configure IAPs for the Expo client -- it's not useful for developers to test their own IAPs.